faunadb-java-dsl/src/test/java/com/faunadb/client/dsl/EnvVariables.java

```diff
@@ -1,7 +1,7 @@
-package com.faunadb.client.test;
+package com.faunadb.client.dsl;
```

Comment:
Can we keep this class on test scope? I wish to not send this with production code.